rosbridge_server/launch/rosbridge_websocket.launch

```diff
@@ -1,16 +1,18 @@
 <launch>
   <arg name="port" default="9090" />
-  <arg name="address" default="" />
+  <arg name="address" default="localhost" />
   <arg name="ssl" default="false" />
   <arg name="certfile" />
   <arg name="keyfile" />
   <arg name="authenticate" default="false" />
+  <arg name="enable_authentication" default="false" />
```

Comment:
Not for now, but in the future we need to think and figure out a better way to configure more than one authentication mechanism at the same time.